server/innernet-server@.service

```diff
@@ -2,12 +2,18 @@
 Description=innernet server for %I
 After=network-online.target nss-lookup.target
 Wants=network-online.target nss-lookup.target
+# Disable systemd's unit start rate limiting logic, which could override Restart=always.
+# See https://unix.stackexchange.com/q/289629/352972
+StartLimitIntervalSec=0
```

Comment:
I'm not so sure we want to modify this unit much at all - if the server fails it's more likely to be a permanent problem and the typical systemd "give up" logic makes more sense, in my opinion.

Personally, I'd suggest we should only slightly increase the `RestartSec` value and leave it otherwise as-is.

Reply:
> I'm not so sure we want to modify this unit much at all - if the server fails it's more likely to be a permanent problem and the typical systemd "give up" logic makes more sense, in my opinion.

That's a good point. I've made changes to systemd units more conservative in a fixup commit to resolve this and the 2 other comments.